design/history/README.md

```diff
@@ -0,0 +1,248 @@
+in discussion, and *not as the discussion format itself*.
+
+Therefore, if you encounter a document in the history directory that you think
+needs substantial iteration and new content, the way to move forward is:
+
+- gather your ideas and start writing
+- prepare to create a new file
+- commit and share your notes, linking to existing content where you can.
+
+If you don't have push access to this repository, or want to start writing and
+sharing even earlier iterations of thoughts, Github Gists are also a great way
+to handle this style of editing.
+This directory can be thought of roughly like a collection of gists; we've just
+gathered the ones especially relevant to the evolution of the design of IPLD.
+
+
+Specific forms of History
+-------------------------
+
+### Exploration Reports
+
+Exploration Reports are one general kind of document we use in order to
+gather thoughts, share early drafts of concepts, and begin to solicit feedback.
+
+Exploration Reports don't have a *strict* format, just general intentions.
+Part of the purpose of exploration reports is to *make writing possible*
+and lower the barriers to entry to get something written and sharable;
+overly strict formats and formalizations would be antithetical to this purpose.
+
+Exploration Reports typically have a *single author*.  The intent behind this
+is to encourage writing things down and forming bodies of thought which
+we can reference in the future -- without putting any burdens of
+consensus-seeking up-front, which we observe can become a barrier that makes
+sharing and recording early design processes harder.
+Correspondingly, Exploration Reports don't need to reach for full solutions,
+which means as long as the report covers reasonably interesting ground,
+it should often be possible to merge them with minimal turnaround time.
+
+You can find similar patterns used in other communities described here:
+
+- https://github.com/golang/go/wiki/ExperienceReports
+
+
+FAQ
+---
+
+### Why not use github issues?
+
+Github issues are a form of "inbox": they represent things that need active response.
+
+Github issues are a medium for discussion: that's useful, but in many ways they
+don't lend themselves to closure, nor do they make it easy to create checkpoints
+or periodic resummarizations to keep dicussions on track; and they definitely
+are not themselves suitable to being treated as artifacts for reference.
+
+There are several reasons we believe a directory of files to be superior to
+github issues for our purposes:
+
+- It's preferable to save github issues for real issues that need active engagement;
+- it's possible to link to specific headings in markdown in files in github,
+  while it is *not* possible to do this to headings within in issue bodies;
+- and it's important and desirable that conversations reach halting states;
+  we want to emphasize that "thread necros" are *bad*, and rebooting a
```

Comment:
This in particular could use better statement.  If you've heard the phrase 'thread necro" before, it unpacks instantly; if you haven't, it's *extremely* inscrutable.

Reply:
I'd really like this one to be rephrased. Perhaps:

> we want to emphasize that reviving discussions that haven't seen any update for a long time are *bad*